src/main/java/org/sagebionetworks/bridge/exporter3/Exporter3WorkerProcessor.java

```diff
@@ -545,4 +654,20 @@ private String getCalendarDateForRecord(HealthDataRecordEx3 record) {
     InputStream getBufferedInputStream(InputStream inputStream) {
         return new BufferedInputStream(inputStream);
     }
+
+
+    protected File extractFileFromZip(java.io.File directory, java.io.File zipFile, String fileName) {
+        try {
+            ZipFile zFile = new ZipFile(zipFile);
+            zFile.extractFile(fileName, directory.getPath());
+            return fileHelper.newFile(directory, fileName);
+        } catch (ZipException e) {
+            LOG.error("Error extracting file from zip", e);
+        }
+        return null;
+    }
```

Comment:
TODO: Since we're unzipping user input, we need to defend against Zip Bomb attacks.

Also, the ZipFile library doesn't seem to play well with our InMemoryFileHelper, so we may need to refactor this.